File: Android/app/src/main/java/com/example/agileus/models/Message.kt
package com.example.agileus.models

import com.google.gson.annotations.SerializedName

data class Message (
    @SerializedName("idEmisor") val idEmisor:String,
    @SerializedName("idReceptor") val idReceptor:String,
    @SerializedName("texto") val texto:String,
    @SerializedName("fechaCreacion") val fechaCreacion:String
)

File: IOS/AgileUs/AgileUs/Tareas/Models/Message.swift
//
//  Message.swift
//  AgileUs
//
//  Created by Carlos nitsuga Hernandez hernandez on 01/12/21.
//

import Foundation

struct MessageTask: Codable {
    var idEmisor: String?
    var idReceptor: String?
    var texto: String?
    var rutaDocumento: String?
    var fechaCreacion: String?
}

struct messageTaskResponse: Codable {
    var status: String?
    var msj: String?
    var data: String?
}
